README.md

```diff
@@ -13,6 +13,13 @@ This is the Helm Chart for [Redpanda](https://vectorized.io).
 
 ## Installation
 
+### Using the repository
+
+```sh
+helm repo add vectorizedio https://vectorizedio.github.io/helm-charts/
```

Comment:
I believe the docs use `redpanda` as the repo name, should we use `redpanda` here or change the docs to be the same?

Reply:
But then it becomes: `helm install redpanda/redpanda` 

vectorizedio matches the url and is closer to `docker run vectorized/redpanda`

And the repository is "helm-charts", so other Vectorized wares could be listed there.

But I hold no strong opinion.

Reply:
Are there docs for this already?

Reply:
Oh, I see `helm repo add redpanda https://charts.vectorized.io/`

Reply:
There are, we can update the docs. https://vectorized.io/docs/quick-start-kubernetes

Reply:
I hold no strong opinions on this! Was just trying to do some tidy up.

Reply:
If the chart is already published, we can probably close this, or link the K8s quick start guide instead. One single mechanism is better.